2020/day-17/arora-aditya/day17.py

```diff
@@ -0,0 +1,231 @@
+from typing import List, Dict, Set
+from collections import *
+from functools import lru_cache
+from math import log10
+from helper.submit.submit import *
+from utils import *
+from pprint import pprint as pp
+import numpy as np
+
+
+DAY = 17
+setup(DAY)
+lines = read_file(DAY)
+
+cubes = [
+    (-1, -1, -1),
+    (-1, -1, 0),
+    (-1, -1, 1),
+    (-1, 0, -1),
+    (-1, 0, 0),
+    (-1, 0, 1),
+    (-1, 1, -1),
+    (-1, 1, 0),
+    (-1, 1, 1),
+    (0, -1, -1),
+    (0, -1, 0),
+    (0, -1, 1),
+    (0, 0, -1),
+    (0, 0, 1),
+    (0, 1, -1),
+    (0, 1, 0),
+    (0, 1, 1),
+    (1, -1, -1),
+    (1, -1, 0),
+    (1, -1, 1),
+    (1, 0, -1),
+    (1, 0, 0),
+    (1, 0, 1),
+    (1, 1, -1),
+    (1, 1, 0),
+    (1, 1, 1),
+]
```

Comment:
Dunno if you generate these, but I made this on the first cellular automata of this year and just reused it:
```python
for xk in range(-1, 2):
    for yk in range(-1, 2):
        for zk in range(-1, 2):
            for wk in range(-1, 2):
                if xk == yk == zk == wk == 0:
                    continue
                ...
```

Reply:
i had them from last year should've just generated